go.mod

```diff
@@ -1,13 +1,11 @@
 module github.com/conduitio/conduit-connector-kafka
 
-go 1.21.1
-
-toolchain go1.22.2
+go 1.22.5
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/conduitio/conduit-commons v0.2.0
-	github.com/conduitio/conduit-connector-sdk v0.9.2-0.20240619124114-6a89e0d7add8
+	github.com/conduitio/conduit-commons v0.2.1-0.20240717151024-0c8d1f406cb2
+	github.com/conduitio/conduit-connector-sdk v0.9.2-0.20240730174041-501732783ac5
```

Comment:
Shall we update to `main`?

Reply:
Looks like it's pointing to https://github.com/ConduitIO/conduit-connector-sdk/commit/501732783ac5.